### DISCUSSION TITLE
支持整张专辑下载

### DISCUSSION THREAD

**Discussion by rong6** [SPlayer-Dev/SPlayer#268] [想法 Ideas]
目前仅支持单首歌曲下载，如图：
![image](https://github.com/user-attachments/assets/2a040433-ed8a-43eb-b60b-3976a7cae0fa)
![image](https://github.com/user-attachments/assets/a2c365fd-a619-41dc-84c4-a2677f97cbbe)

希望能增加下载整张专辑所有音乐：
![image](https://github.com/user-attachments/assets/b4edae98-f3e4-4439-8722-ad0572e38128)

**Comment by imsyy** [comment]
下载功能并非本软件主要功能，目前可能暂时不会考虑添加批量下载功能

**Comment by rong6** [comment]
> 下载功能并非本软件主要功能，目前可能暂时不会考虑添加批量下载功能

理解，但既然可以作为本地播放器，下载后再在本地播放（就是本地缓存嘛）我觉得无可厚非。如果可以的话还是可以加到ToDo里，优先级低一点没关系( ͡° ͜ʖ ͡°)✧
